src/Libraries/DSOffice/OpenXmlHelper.cs

```diff
@@ -345,7 +345,7 @@ private static object GetCellValue(Cell cell, SharedStringTable sharedStringTabl
                     else
                     {
                         // This is a number
-                        if (cell.CellValue.TryGetDouble(out var value))
+                        if (double.TryParse(cell.CellValue.InnerText, NumberStyles.Any, NumberFormatInfo.InvariantInfo, out var value))
```

Comment:
Have we tested change in behavior when digit separator is changed (from OS settings)?

Reply:
Possible use case: customer distributes a spreadsheet with per-cell number formatting to at least 2 different users in 2 different locations with different OS settings relating to number formatting (digit separator). Did the previous function work for both OS settings and does the new one not work the same?